### PR TITLE
This adds ProphetStor Federator.ai Recommended Monitors.

### PR DESCRIPTION
**Pull request by chenyingtz**
### What does this PR do?

This adds ProphetStor Federator.ai Recommended Monitors.

### Motivation

add Federator.ai Recommended Monitor for alerting long latency in Kafka queue

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by hithwen** [comment]
Hallo @chenyingtz, sorry for the trouble but could you update your PR with latest master? That should solve the failing tests issue

File: federatorai/assets/recommended_monitors/federatorai_kafka_queue_latency.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		"thresholds": {
+			"critical": 1,
+			"critical_recovery": 0
+		},
+		"threshold_windows": {
+			"trigger_window": "last_15m",
+			"recovery_window": "last_15m"
+		}
+	}
+}
```
Comment (ChristineTChen) [inline federatorai/assets/recommended_monitors/federatorai_kafka_queue_latency.json:29]:
```suggestion
		}
	},
	
	"recommended_monitor_metadata": {
		"description": "Notify your team when messages in the Kafka topic takes longer than the usual time to be processed."
	}
}
```

Let's also include this field

Comment (chenyingtz) [reply federatorai/assets/recommended_monitors/federatorai_kafka_queue_latency.json]: thanks! already added this field to federatorai_kafka_queue_latency.json

File: federatorai/assets/recommended_monitors/federatorai_kafka_queue_latency.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+	"id": 20550435,
+	"name": "Long Latency in Kafka Queue",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(( ( sum:kafka.consumer_lag{*} by {kube_cluster,consumer_group,topic} / sum:federatorai.kafka.consumer_offset_rate{*} by {kube_cluster,consumer_group,topic} ) / sum:datadog.agent.running{*} by {kube_cluster} ) * 60 * 1000, 'basic', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true') >= 1",
```
Comment (sanjuashok) [inline federatorai/assets/recommended_monitors/federatorai_kafka_queue_latency.json:5]: Using the general metric here may not be a good idea.

**Pull request merged**